### PR TITLE
[INDY-505] System tests for CD pipeline

### PR DESCRIPTION
**Pull request by andkononykhin**
Continues #998

### CODE REVIEW

**Comment by andkononykhin** [comment]
test this please

**Comment by andkononykhin** [comment]
test this please

File: Jenkinsfile.cd [hunk 1]
```diff
@@ -64,8 +64,124 @@ def buildDebUbuntu = { repoName, releaseVersion, sourcePath ->
...
+def systemTests = { component ->
+
+    String prefix = "System Tests ($component)"
+    String defaultNetwork = 'pool-network'
+
+    def dockerClean = { networkName ->
+        networkName = networkName ?: defaultNetwork
+
+        try {
+            sh "docker ps -q --filter network=$networkName | xargs -r docker rm -f"
```
Comment (anikitinDSR) [inline Jenkinsfile.cd]: What if container is not stopped yet?

Comment (andkononykhin) [reply Jenkinsfile.cd]: `rm -f`, we don't care

**Pull request merged**